package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "queue",
+  "version": "1.0.0",
+  "description": "simple job queue",
+  "main": "index.js",
+  "engine": {},
+  "scripts": {
+    "test": "NODE_ENV=test tap --cov test/*.test.js"
+  },
+  "author": "First+Third",
+  "license": "MIT",
+  "devDependencies": {
+    "eslint": "^4.17.0",
+    "eslint-config-firstandthird": "^4.3.0",
+    "eslint-plugin-import": "^2.8.0",
+    "tap": "^11.1.1"
+  },
+  "eslintConfig": {
+    "extends": "firstandthird"
+  },
+  "//": "Using older mongodb because of this: https://github.com/Marsup/hapi-mongodb/issues/38",
```

Comment:
eh? we aren't using hapi here

Reply:
the 3.0.0 mongo driver has some weird quirks, looked at how they were handling it since I know that worked with promises.